homeassistant/helpers/trigger_template_entity.py

```diff
@@ -389,3 +391,28 @@ def __init__(
         ManualTriggerEntity.__init__(self, hass, config)
         self._attr_native_unit_of_measurement = config.get(CONF_UNIT_OF_MEASUREMENT)
         self._attr_state_class = config.get(CONF_STATE_CLASS)
+
+    @callback
+    def _set_value_with_possible_timestamp(
+        self,
+        value: Any,
+        device_class: SensorDeviceClass | None,
+        variables: dict[str, Any],
+    ) -> None:
+        """Set value with possible timestamp.
+
+        Implementing class should call this last in update method to set the value
+        and then render all templates.
```

Comment:
I don't understand what this means, I read it as the implementing classes should first call this method, and then do something else to render templates, but the implementing classes in this PR does not do that?

Reply:
Should be ok now when `_process_...` is not included anymore